providers/openmetrics/server_test.go

```diff
@@ -292,6 +293,7 @@ func toFloat64HistCount(h prometheus.Observer) uint64 {
 }
 
 func requireValue(t *testing.T, expect int, c prometheus.Collector) {
+	t.Helper()
```

Comment:
That way error traces when a test fails lead to the line where this function was invoked, not into this function. Convenience thing, helped me to navigate failing tests.